bugs/index.md

```diff
@@ -27,6 +27,15 @@ triaged by Chrome and WebRTC engineers.
 Please include as many relevant details as possible.
 
 
+### Filing a Security Bug
+
+The WebRTC team takes security very seriously. If you find a vulnerability in
+WebRTC, please file a [Chromium security bug][ChromeSecurity], even if the bug
+only affects native WebRTC code and not Chromium.
+
+Any vulnerabilities will be disclosed on webrtc-discuss once we have fixed the
```

Comment:
discuss-webrtc?

Maybe a link to the group as well? https://groups.google.com/group/discuss-webrtc

Reply:
Good catch, thanks